packages/fast-components-react-base/src/context-menu-item/context-menu-item.props.ts

```diff
@@ -25,6 +25,12 @@ export interface ContextMenuItemHandledProps extends ContextMenuItemManagedClass
      * If the menu item is disabled
      */
     disabled?: boolean;
+
+    /**
+     * Callback for when an item is invoked
+     * Returns the prop contract for the invoked context menu item
+     */
+    onInvoke?: (props: ContextMenuItemProps) => void;

```

Comment:
why would you want all of the props for that context menu item?

Reply:
Great question @janechu. Pinging @nicholasrice for awareness as he was part of this investigation. 

The idea was to keep this from being a breaking change. By passing back the full prop object, we can pass back the unhandledProps which include an id and other identifiers to know which menu item was invoked. If we pass back an "id" we need to add it to the prop contract and likely require it, which breaks the API

Reply:
In that case what we have been doing is passing a React synthetic event which should also have all of this information, plus the event information.

Reply:
For transparency, synced offline and we'll pass back both the event object AND the props for the item